builder/llpcBuilderImplMisc.cpp

```diff
@@ -31,8 +31,9 @@
 #include "llpcBuilderImpl.h"
 #include "llpcContext.h"
 
-#include "llvm/IR/Intrinsics.h"
 #include "llvm/IR/InlineAsm.h"
+#include "llvm/IR/Intrinsics.h"
+#include "llvm/IR/IntrinsicsAMDGPU.h"
```

Comment:
Do you need both of these? Or does IntrinsicsAMDGPU.h include Intrinsics.h?

Reply:
Good point. IntrinsicsAMDGPU.h does not include Intrinsics.h, but I do not need to include Intrinsics.h, if an impl file only references Intrinsic::amdgcn_* intrinsics (like here). I will simplify.